packages/protocol/contracts/governance/Proposals.sol

```diff
@@ -235,16 +235,19 @@ library Proposals {
    * @return The stage of the dequeued proposal.
    * @dev Must be called on a dequeued proposal.
    */
-  function getDequeuedStage(Proposal storage proposal, StageDurations storage stageDurations)
-    internal
-    view
-    returns (Stage)
-  {
+  function getDequeuedStage(
+    Proposal storage proposal,
+    StageDurations storage stageDurations,
+    bool expireProposalWithoutTransactions
```

Comment:
Why exactly do we need two different behaviors from this function?

Reply:
We don't - I misunderstood the task a little bit - removed